shell/app/modules/msp/monitor/trace-insight/pages/trace-querier/trace-graph/index.tsx

```diff
@@ -427,10 +427,14 @@ export function TraceGraph(props: IProps) {
         )}
 
         {view === 'flame' && (
-          <div ref={containerRef} className="relative">
+          <div
+            ref={containerRef}
+            className="relative graph-flame overflow-y-auto overflow-x-hidden"
+            style={{ height: window.innerHeight - 200 }}
+          >
             <FlameGraph
               data={formatFlameData()}
-              height={200}
+              height={20 * dataSource?.depth + 1}
```

Comment:
in case dataSource is undefined, better be `dataSource ? 20 * dataSource.depth + 1 : 200 `